packages/cactus-plugin-consortium-manual/src/main/typescript/plugin-consortium-manual.ts

```diff
@@ -216,4 +227,46 @@ export class PluginConsortiumManual
   public getAspect(): PluginAspect {
     return PluginAspect.CONSORTIUM;
   }
+
+  public async createJws(): Promise<JWSGeneral> {
+    const { keyPairPem, consortiumRepo: repo } = this.options;
+
+    this.updateMetricNodeCount();
+    const keyPair = JWK.asKey(keyPairPem);
+    const payloadObject = { consortiumDatabase: repo.consortiumDatabase };
+    const payloadJson = jsonStableStringify(payloadObject);
+    const _protected = {
+      iat: Date.now(),
+      jti: uuidv4(),
+      iss: "Hyperledger Cactus",
+    };
+    // TODO: double check if this casting is safe (it is supposed to be)
+    return JWS.sign.general(payloadJson, keyPair, _protected) as JWSGeneral;
+  }
+
+  public async getJws(): Promise<JWSGeneral> {
```

Comment:
@AzaharaC Please name this method to match the endpoint that it is the backing implementation of, e.g. something like `getConsortiumJws()`